File: hw_06/Python/hw_06/geometry/Circle.py
import math

class Circle:
    def __init__(self, radius:float=1.0):
        self.__radius = radius

    def get_radius(self):
        return self.__radius
    
    def set_radius(self, radius:float):
        self.__radius = radius

    def area(self):
        return math.pi * pow(self.__radius, 2)
    
    
    def circle_info(self):

        return f"Circle: Radius = {self.__radius}, Area = {self.area():.2f}"

File: hw_06/CPP/cpp_oppwork06/geometry/circle.cpp
#include <iostream>
#include <cmath>
#include <iomanip>

using namespace std;

class Circle {
    private: 
        double radius;

    public:
        double getRadius(){
            return radius;
        }

        void setRadius(double radius){
            this->radius = radius;
        }

        Circle(double radius): radius(radius){}

        Circle():Circle(6.0){}

        double area(){
            return M_PI * pow(radius, 2);
        }

        void cirInfo(){
            cout << fixed << setprecision(2) << "Circle: Radius = " << radius << ", Area = " << area() << endl;
        }   

};
